plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -158,9 +163,12 @@ export function createPublishGithubAction(options: {
 
       const remoteUrl = data.clone_url;
       const repoContentsUrl = `${data.html_url}/blob/master`;
+      const outputPath = ctx.input.sourcePath
+        ? resolvePath(ctx.workspacePath, ctx.input.sourcePath)
```

Comment:
We need to check so that the resulting dir is within the workspace, otherwise you could do some a bit too fun things with this action.

Reply:
We should probably check this in the Cookiecutter plugin as well  - that's where this logic was inspired from :)